config/django/base.py

```diff
@@ -120,7 +125,7 @@
 
 LANGUAGE_CODE = "en-us"
 
-TIME_ZONE: str = env("TIME_ZONE", default="UTC")
+TIME_ZONE = env("TIME_ZONE", default="UTC")

```

Comment:
```suggestion
TIME_ZONE: str = env("TIME_ZONE", default="UTC")

```

Every global variables is supposed to have typed at every declaration!